thirdeye-worker/src/main/java/ai/startree/thirdeye/worker/task/runner/NotificationTaskRunner.java

```diff
@@ -102,9 +93,11 @@ public List<TaskResult> execute(final TaskInfo taskInfo, final TaskContext taskC
   }
 
   public List<TaskResult> execute(final long subscriptionGroupId) throws Exception {
+    final SubscriptionGroupDTO sg = getSubscriptionGroupDTO(subscriptionGroupId);
```

Comment:
this changes the timer semantic, before fetching the subscription group was part of the time measurement

I think this is fine but NOTIFICATION_TASK_TIMER_DESCRIPTION needs to be updated

Reply:
edit: 
actually I think this is an issue. The current measurement is what matters. 
we should be able to get the original namespace from the taskDto. 
this may require changes: 
either by adding namespace to TaskInfo, either by adding namespace as an argument to TaskRunner#execute

Reply:
same issue with the change in DetectionPipelineTaskRunner

Reply:
Done. resolved both